drivers/lsm303dlhc/lsm303dlhc.c

```diff
@@ -45,20 +45,17 @@ int lsm303dlhc_init(lsm303dlhc_t *dev, const lsm303dlhc_params_t *params)
     /* Acquire exclusive access to the bus. */
     i2c_acquire(DEV_I2C);
 
-    DEBUG("lsm303dlhc reboot...");
+    DEBUG("lsm303dlhc init...");
```

Comment:
Minor nit, but would it make sense to place this debug before the i2c_aqcuire (to reduce the time that i2c is locked)?

Reply:
@fjmolinas wait before merging, even if minor, I agree with this suggestion.

Reply:
Addressed. Thanks for the suggestion @wosym.